envault/cli.py

```diff
@@ -33,13 +35,56 @@ def get_secrets(server, secret, token):
 def cli():
     pass
 
+@cli.command("init")
+@click.option("-profile", help="profile for the config")
```

Comment:
lets remove profile option from here and add it to the cli itself where user is prompted to input the profile